lib/orderbook/OrderBook.ts

```diff
@@ -79,7 +79,8 @@ class OrderBook extends EventEmitter {
   private bindPool = () => {
     if (this.pool) {
       this.pool.on('packet.order', this.addPeerOrder);
-      this.pool.on('packet.orderInvalidation', order => this.removePeerOrder(order.orderId, order.pairId, order.quantity));
+      this.pool.on('packet.orderInvalidation', order =>
+        this.removePeerOrder(order.orderId, order.pairId, order.peerPubKey, order.quantity));
```

Comment:
We'd probably want a try/catch here, and the TODO for penalizing the peer should be in the case we get an unauthorized error.